src/core/Form/TextInput/TextInput.tsx

```diff
@@ -33,11 +39,35 @@ export const textInputClassNames = {
   inputElement: `${baseClassName}_input`,
   styleWrapper: `${baseClassName}_wrapper`,
   statusTextHasContent: `${baseClassName}_statusText--has-content`,
+  bottomWrapper: `${baseClassName}_bottomWrapper`,
```

Comment:
Class names are in kebab-case so bottom-wrapper & character-counter, character-counter--error

Reply:
Well spotted. I'll change these.